recipes/libsndfile/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.0.26" %}
+
+package:
+  name: libsndfile
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erikd/libsndfile.git
+  git_tag: f67f0992eae3d19ca3a3804c8ea6b081b2131e5f
```

Comment:
Can we download a tarball instead? They have some on their [site]( http://www.mega-nerd.com/libsndfile/#Download ).